crates/ruma-events-macros/src/event_enum.rs

```diff
@@ -884,7 +928,13 @@ fn to_event_content_path(
         .collect();
 
     let content_str = match kind {
+        EventKind::ToDevice if prefix.is_some() => {
+            format_ident!("{}{}ToDeviceEventContent", prefix.unwrap(), event)
+        }
         EventKind::ToDevice => format_ident!("{}ToDeviceEventContent", event),
```

Comment:
This can just be one branch, no?

```suggestion
        EventKind::ToDevice => format_ident!("{}{}ToDeviceEventContent", prefix.unwrap_or(""), event),
```